pwa/src/templates/templateParts/resultsTemplates/layers/LayersResultTemplate.module.css

```diff
@@ -0,0 +1,41 @@
+.Container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.Container > * {
+  margin-block-end: var(--nlportal-space-block-lg);
+}
+
+.Container > *:not(:last-child) {
+  margin-inline-end: var(--nlportal-space-block-xl);
+}
+
+.Components {
+  display: inline-block;
+  background-color: rgb(245, 245, 245);
+  box-shadow: rgb(33 33 33 / 25%) 0px -3px 0px inset;
+  padding: 14px;
+}
+
+.Components > a {
+  text-decoration: none;
+}
```

Comment:
If we decide to remove the `Link` element around the name, these rules would become redundant.